content/en/platform/corda/5.0/developing-applications/external-messaging.md

```diff
@@ -0,0 +1,78 @@
+---
+date: '2023-06-22'
+title: "External Messaging CorDapps"
+project: corda
+version: 'Corda 5.0 Beta 4'
+menu:
+  corda5:
+    identifier: corda5-develop-external-messaging
+    parent: corda5-develop
+    weight: 7050
+section_menu: corda5
+---
+
+# External Messaging CorDapps
+
+A running Corda flow can send simple messages via Kafka to external systems. 
+In 5.0, this is limited to sending messages, but a future version will support both send and send-and-receive messages. 
+
+External messaging is implemented in CorDapps by the following components:
+* **Channels** - abstract representations of routes from a flow to an external system. They allow Cluster Aministrators and Network Operators to control the Kafka implementation of a logical channel at the cluster and virtual node level. A CorDapp Developer is responsible for defining the channels as part of the CorDapp.
+* **Routes** - a per virtual node configuration of the channel and its behaviour for the specific virtual node. This includes the actual Kafka topic to be used, if the route is active or not, and how the flow API responds to an inactive route.
+* **Default Route Configuration** - the route configuration used, along with any channels defined in the CorDapp, to generate the virtual node’s routes. The default route configuration is defined at the cluster level and can be updated via the <a href="../../reference/rest-api/C5_OpenAPI.html#tag/Configuration-API/operation/put_config">`config` endpoint</a> of the REST API. For more information see [Configuring External Messaging]({{< relref "../deploying-operating/external-messaging/_index.md">}}).
```

Comment:
The hyperlink is not working. I am getting an error message. It is the same as the other one. Please see the other comment in this PR.

Reply:
This is the error message:
![image](https://github.com/corda/corda-docs-portal/assets/6329837/a8a0a910-737e-4515-8f43-e67cde9f4ff9)